recipes/xmscore/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "xmscore" %}
+{% set version = "3.0.7" %}
+{% set sha256 = "98acd332ac5a14602539e218db82b2970dbeed2b149e0d47dcbd80c6778801c7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/Aquaveo/{{ name|lower }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  string: py{{ python }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.0.0
+    - boost =1.67
+    - pybind11 =2.2.4
+    - python {{ python }}
+  host:
+    - python {{ python }}
```

Comment:
Please remove the python version jinja and let the conda-forge configuration handle the build matrix. If this is a `Python 3.6` only package you can do that by specifying `>=3.6,<3.7` here.